### DISCUSSION TITLE
v11的qq如何私聊发送图片

### DISCUSSION THREAD

**Discussion by paul-sama** [nonebot/discussions#386] [问答]
```

from nonebot.adapters import Event
from nonebot.adapters.onebot.v11 import Bot
from nonebot.adapters.onebot.v11.message import MessageSegment

matcher_random_weapon = on_command('随机武器')

@matcher_random_weapon.handle()
async def _(bot: Bot, event: Event):
    await matcher_random_weapon.send(
        MessageSegment.image(
            file=get_random_weapon(),
            cache=False
        )
    )
```

群聊里能正常接受到图片
私聊机器人就什么都看不到
聊天框空白，列表里显示[图片]
<img width="519" alt="image" src="https://github.com/nonebot/discussions/assets/350651/0d753e2f-844e-4530-b834-bac9bcd591f1">

群聊
<img width="407" alt="image" src="https://github.com/nonebot/discussions/assets/350651/ca907638-8a68-45c6-9ed8-852a1d99543a">

私聊
![87e1edbec1a79ff621aac7efa27ec80a](https://github.com/nonebot/discussions/assets/350651/04ab80d2-e619-43cc-a608-eb3448b1b8c8)

**Comment by yanyongyu** [comment]
就是直接send image和群一样，发不出去或者看不见不是nb的问题，你应该问协议端gocq
